src/quicklist.c

```diff
@@ -1715,7 +1716,8 @@ int quicklistTest(int argc, char *argv[]) {
     size_t option_count = sizeof(options) / sizeof(*options);
     long long runtime[option_count];
 
-    for (int _i = 0; _i < (int)option_count; _i++) {
+    iteration = accurate ? (int)option_count : 1;
+    for (int _i = 0; _i < iteration; _i++) {
         printf("Testing Option %d\n", options[_i]);
```

Comment:
setting `iteration` to 0 means we don't test compression, right?
would limiting it to [0,1.2] be too slow?
```suggestion
    iteration = accurate ? (int)option_count : 3;
    for (int _i = 0; _i < iteration; _i++) {
        printf("Testing Compression option %d\n", options[_i]);
```

Reply:
setting iteration to 0 means all tests are not run. I'll test how much the speed has changed.

Reply:
ohh, sorry, for some reason i thought we run only one iteration (the first one with no compression).
(not enough coffee in my blood).
i don't think we want to completely skip this test, we i think we need to run at least with and without compression), and find other ways to tone down the prints and time.

Reply:
I try it.